src/scene/scene_forwardrenderer.js

```diff
@@ -631,7 +636,31 @@ pc.extend(pc, function () {
             drawCalls.sort(sortDrawCalls);
 
             // Render a depth target if the camera has one assigned
-            if (camera._depthTarget) {
+            if (camera._renderDepthRequests) {

```

Comment:
Again, leading _ should indicate that a member is for private internal use.

We shouldn't be accessing them outside of the class.